cppcache/integration-test/ThinClientPutAll.hpp

```diff
@@ -401,7 +401,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, StepThree)
           PdxTests::Address::createDeserializable);
       serializationRegistry->addPdxSerializableType(
           PdxTests::PdxTypes3::createDeserializable);
-    } catch (Exception e) {
+    } catch (Exception &e) {
```

Comment:
Style guide says justify the `&` consistent with the rest of the file. In this case it should be next to the type and not the variable.